rackcity/utils/asset_utils.py

```diff
@@ -285,9 +286,7 @@ def save_network_connections(asset_data, asset_id, change_plan=None):
             failure_message += "Port name '" + port_name + "' is not valid. "
             continue
         if not network_connection_data_has_destination(network_connection):
-            # Delete network connection
-            if change_plan:
-                handle_network_connection_delete_on_cp(port_name, asset_id, change_plan)
+
```

Comment:
this was causing a lot of uncaught errors-> commenting this out is actually resulting in behavior that is more expected... still not sure if we fully handle network connection deletions but think its better to just not attempt to handle it if we have uncaught errors

Reply:
haha. ok